sopel_modules/chanlogs/__init__.py

```diff
@@ -54,7 +54,7 @@ class ChanlogsSection(StaticSection):
     quit_template = ValidatedAttribute('quit_template', default=None)
     nick_template = ValidatedAttribute('nick_template', default=None)
     topic_template = ValidatedAttribute('topic_template', default=None)
-
+    opt_in_channels = ListAttribute('opt_in_channels')
```

Comment:
There shouldn't be a deleted blank line; flake8 (if it was set up for this repo) would fail with only one blank line between this and the following function def.